### PR TITLE
Add support for import() expressions

### PR DESCRIPTION
**Pull request by haggholm**
Add support for lazy `import()` calls as used by `System.import()` and lazy Webpack chunks.

Modelled on the `require.resolve()` PR (https://github.com/depcheck/depcheck/pull/196). Not too familiar with the test framework, but copying tests, it seems to work (and mutating tests does properly break…).

Perhaps this should be optional/disabled by default, but the options (esp. w.r.t. the tests) are a bit opaque to me: feedback extremely welcome.

### CODE REVIEW

File: test/fake_modules/system_import_missing/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+System.import('anyone');
```
Comment (lijunle) [inline test/fake_modules/system_import_missing/index.js]: I suppose the `System.import` has been deprecated and we don't need to support it.

File: test/fake_modules/import_function/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+import('optimist');
```
Comment (lijunle) [inline test/fake_modules/import_function/index.js]: Could you please try the webpack named chunk feature (aka, the magic comment):

```js
import(/* webpackChunkName: 'chunk' */ 'chunk')
```

**Comment by haggholm** [comment]
I guess I was about as quick to reply as you were! I’m not 100% what you mean; do these changes reflect what you wanted?

**Comment by lijunle** [comment]
@haggholm Webpack's `import()` has a feature named [*magic comment*](https://medium.com/webpack/webpack-3-official-release-15fd2dd8f07b#ea65) to specify the chunk name with the following syntax:

```js
import(/* webpackChunkName: "my-chunk-name" */ 'module');
```

We should check if we are supporting it.

**Comment by haggholm** [comment]
Right. I added a test that verifies that it still detects the import­ -- I think that’s what you meant? I also cleaned up some cruft I’d accidentally introduced.

**Comment by mnkhouri** [comment]
Thank you for the contribution, @haggholm !  🎉

**Pull request merged**